chain/network/src/network_protocol/mod.rs

```diff
@@ -476,6 +476,8 @@ pub struct RoutedMessageV2 {
     pub msg: RoutedMessage,
     /// The time the Routed message was created by `author`.
     pub created_at: Option<time::Utc>,
+    /// Number of peers this routed message travelled through. Doesn't include the peer that created the message.
```

Comment:
is peer who received the message included?

Reply:
No, which I think is ok.
Will update the comment.